puppet/zulip/manifests/dockervoyager.pp

```diff
@@ -0,0 +1,32 @@
+class zulip::dockervoyager {
```

Comment:
Can you add a brief comment at the top explaining a bit more about what this is?